sdk/storage/Azure.Storage.DataMovement.Blobs/src/DataMovementBlobsExtensions.cs

```diff
@@ -482,6 +484,7 @@ internal static PageBlobCreateOptions GetCreateOptions(
                     TagConditions = options?.DestinationConditions?.TagConditions,
                     LeaseId = options?.DestinationConditions?.LeaseId,
                 },
+                PremiumPageBlobAccessTier = GetAccessTier(options, sourceProperties?.RawProperties).ToPremiumPageBlobAccessTier(),
```

Comment:
You already have this value above but you calculate it again

Reply:
removed the line above